### DISCUSSION TITLE
If you have Quick Switcher++ can i disable the switch core plugin?

### DISCUSSION THREAD

**Discussion by pubdigital** [darlal/obsidian-switcher-plus#140] [Q&A] [answered]
Thanks a lot for this wonderful plugin. I try to unclutter my obsidian any time if possible. I don't find in the front page if obsidian core swith plugin is a dependency for this. I read if an enhance but i wonder if it is possible to disable the core plugin. Thans a lot in advance.

**Comment by darlal** [comment, marked as answer]
Hey, the core switcher plugin is a dependency, the core settings and some functionality are reused, so it should stay enabled. You can unbind the core switcher hotkeys and map them to Switcher++ instead and it should work as expected. Thanks!

**Reply by Snowboy801** [reply]
Better to write it into "Readme". 
I ever thought Switcher++ so powerful and disabled the core switcher, too. 
And then, I found I still need two functions: "only search file name“ and "create a new file" without searching result. So I enabled core switcher and map Ctrl+O to QS++ standard mode. :)
Thank you for your handy plugin!
